src/ArtistList.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import ArtistDetail from './ArtistDetail';
+import Artist from './Artist';
+
+class ArtistList extends React.Component{
+    state ={
+        sortType: '',
+        artists: []
+    }
+
+    componentDidMount() {
+        this.setState({
+            artists: ArtistDetail
+        })
+    }
+
+
+    //Vote handler
+    voteHandler = (artistId) => {
+        const voteArtist = this.state.artists.map(artist => {
+            if(artistId === artist.id) {
+                return Object.assign({}, artist, {
+                    votes: artist.votes + 1
+                })
+            }
+            else {
+                return artist
+            }
```

Comment:
This isn't necessary. If return part is omitted, the original entity is automatically returned. That means `artist` is automatically returned.